StereoKit/Math/Matrix.cs

```diff
@@ -444,6 +444,32 @@ public static Matrix Perspective(Vec2 imageResolution, Vec2 focalLength, float n
 			return Matrix4x4.CreatePerspective(nearPlaneDimensions.x, nearPlaneDimensions.y, nearClip, farClip);
 		}
 		
+		/// <summary>This creates a matrix used for projecting 3D geometry
+		/// onto a 2D surface for rasterization. With the known camera 
+		/// intrinsics, you can replicate its perspective!</summary>
+		/// <param name="imageResolution">The resolution of the image. This
+		/// should be the image's width and height in pixels.</param>
+		/// <param name="focalLength">The focal length of the camera in pixels,
```

Comment:
I'm also a little uncertain what focal length in pixels means, could you elaborate on this a bit? I would understand degrees or millimeters here, but pixels is a bit baffling!

Reply:
Ohh yep, it's in millimeters. I copied the mistake from the other comment so I'll update both

Reply:
Millimeters also has some wiggle room, it's usually dependent on sensor size, so I guess it may also be nice to clarify that a bit too then?

Reply:
Sorry just double checked - actually it is in pixels so the original description is correct. The focal length should be in the same unit as the image resolution for `Vec2 nearPlaneDimensions = imageResolution / focalLength * nearClip;` to work. In OpenCV as well, the camera matrix includes a focal length and principal point in pixel units. You only need to know the sensor size if you convert from pixels -> mm & vice versa